test/FsAutoComplete.Tests.Lsp/CoreTests.fs

```diff
@@ -260,7 +263,7 @@ let tooltipTests state =
           verifySignature 1 5 "val listOfTuples : list<int * int>" // verify we default to prefix-generics style
           verifySignature 2 5 "val listOfStructTuples : list<struct (int * int)>" // verify we render struct tuples in a round-tripabble format
           verifySignature 3 5 "val floatThatShouldHaveGenericReportedInTooltip : float" // verify we strip <MeasureOne> measure annotations
-          verifyDescription
+          pverifyDescription "This test depends on FSharp.Core.xml being in the bin directory of these tests"
```

Comment:
it's trying to load the xml file from `test\FsAutoComplete.Tests.Lsp\bin\Debug\net7.0 `where F`Sharp.Core.dll` is. The xml file isn't there because we have `content:none` turned on for `FSharp.Core`.  Not really sure how to handle this.

Reply:
Removing `content: none` is fine I guess, or removing it only in the scope of the tests via the tests' `paket.references`.